File: js/src/test/base/test.sum.js
// AUTO_TRANSPILE_ENABLED
import ccxt from '../../../ccxt.js';
import testSharedMethods from '../Exchange/base/test.sharedMethods.js';
function testSum() {
    const exchange = new ccxt.Exchange({
        'id': 'sampleexchange',
    });
    // testSharedMethods.assertDeepEqual (exchange, undefined, 'testSum', exchange.sum (), undefined); // todo: bugs in py
    testSharedMethods.assertDeepEqual(exchange, undefined, 'testSum', exchange.sum(2), 2);
    testSharedMethods.assertDeepEqual(exchange, undefined, 'testSum', exchange.sum(2, 30, 400), 432);
    // testSharedMethods.assertDeepEqual (exchange, undefined, 'testSum', exchange.sum (2, undefined, [ 88 ], 30, '7', 400, null), 432); // todo: bugs in php
}
export default testSum;

File: julia/Ccxt/test/base/test.sum.jl
using Test
using Ccxt
function testSum()

    exchange = get(ccxt, Symbol("Exchange"), nothing)(Dict{Symbol, Any}(
        Symbol("id") => "sampleexchange"
    ));
    assertDeepEqual(testSharedMethods, exchange, nothing, "testSum", sum(exchange, 2), 2);
    assertDeepEqual(testSharedMethods, exchange, nothing, "testSum", sum(exchange, 2, 30, 400), 432);
end
